lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -158,6 +158,33 @@ function isObjectLocked(bucket, objectMD, headers) {
     return false;
 }
 
+
+/* objectLockRequiresBypass will return true if the retention info change
+ * would require a bypass governance flag to be true.
+ * In order for this to be true the action must be valid as well, so going from
```

Comment:
should action validity be in its own function? Or this function renamed to reflect its checking action validity in addition to the bypass governance flag?